src/locales/de/pokemon.json

```diff
@@ -0,0 +1,902 @@
+{
+  "alt": {
+    "mega-sep": "-"
+  },
```

Comment:
Maybe we should find a way of telling the translators what is missing? Currently, they need to cross reference the english translations with their translations, to see what they are missing to fully translate it.
I would suggest that we for example add all the 'alt'-translations to each language file, but just keep the english translation for each. Then translators from the different countries just need to open their language file, and scan it for english words.

Each time we add new translations in the future, we will then add the key-value-pair to each language-file.

Reply:
There are plugins for i18next or even crowdsourced translation platforms that will scan our translation files and report to people which things are missing, but I'm leaving that for when we move translations into a separate repo.

For now just copying over the English versions into each would be ok.

Reply:
I've added (i18next-locales-sync)[https://github.com/felixmosh/i18next-locales-sync] to the translation repo for making sure all language files have the latest translation keys. It will fill them in with an empty string, so it is obvious which things have not been translated yet.

It is there as an npm script for people adding new things, and also in a github workflow run on pull requests to make sure it has been done.